test/codeGenerationTest.ts

```diff
@@ -294,6 +299,10 @@ namespace GenerateConfig {
 				SORTING_TYPE.ALPHABETICAL,
 				SORTING_TYPE.LINE_LENGTH,
 			]),
+			sortingOrder: Util.randomItem([
+				SORTING_ORDER.ASCENDING,
+				SORTING_ORDER.DESCENDING,
+			]),
```

Comment:
I should have done this myself before but can you turn this into `Util.randomItem(Object.values(SORTING_ORDER))`? Then all cases are covered. Same goes for the line above that I should have done myself ;)